ArcadiaParties.API/CustomMiddlewares/DatabaseRolesMiddleware.cs

```diff
@@ -27,7 +27,7 @@ public async Task InvokeAsync(HttpContext context, IMediator mediator)
 
             newIdentity.AddClaim(new Claim(ClaimTypes.Name, user.Identity.Name));
 
-            var query = new GetUserRolesQuery(user.Identity.Name);
+            var query = new GetUserRolesQuery(context.User);
```

Comment:
You already wrote `var user = context.User` before. Why do you need to do this twice?

Reply:
I've fixed it